docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -435,6 +435,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
+------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "4/5ths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
```

Comment:
I would add a line here that says that this 4/5th rule originates from US labor law.

Reply:
Mentioned (slightly further down).... of course, calling it 'labor' implicitly means it's 'US' ;-)

Reply:
The intended the emphasis was on *here*! Mostly related to my next comment re. differentiation between legal definitions of discrimination and "fairness".

Reply:
PS. In an unexpected turn of events there are legal systems outside of the Anglosphere (what?!), but I suppose "arbeidsrecht" won't be very understandable for most of our readers ;)